Sources/Data/models/PlaylistFolder.swift

```diff
@@ -118,13 +118,23 @@ final public class PlaylistFolder: NSManagedObject, CRUD, Identifiable {
       playlistFolder.creatorLink = folder.creatorLink
       
       PlaylistFolder.reorderItems(context: context)
+      
+      let policy = context.mergePolicy
+      context.mergePolicy = NSMergePolicy(merge: .mergeByPropertyObjectTrumpMergePolicyType)
       PlaylistFolder.saveContext(context)
+      context.mergePolicy = policy
       
       DispatchQueue.main.async {
         completion?(folderId)
       }
     }
   }
+  
+  public static func clearMemoryContext() {
```

Comment:
This doesn't seem to be called anywhere?

Reply:
Yes forgot to remove this function, this is not necessary